README.md

```diff
@@ -54,9 +54,24 @@ yarn install
 
 CEA Devs, see the ForumCredentials repository for access to a remote dev database. Otherwise, do the following:
 
-TODO: Postgres instructions (but basically: run a postgres db)
+Run a local postgres instance, version 15. For example, if you're on macos:
 
-(Then there's probably something about migrating, see Database Migrations)
+```bash
+brew install postgresql@15
+brew services start postgresql@15
+```
+
+Configure the schema:
+
+```bash
+psql -f ./schema/accepted_schema.sql
+```
```

Comment:
This didn't quire work for me as written, because it tried to connect to a database that didn't exist (using the name of my laptop as the db name).
``````suggestion
Create the database:

```
psql postgres://localhost:5432/postgres
```
Then in the psql prompt
```sql
CREATE DATABASE eaforum;
\q -- exit psql
```

Configure the schema:

```bash
psql postgres://localhost:5432/eaforum -a -f ./schema/accepted_schema.sql
```
``````

Reply:
Thanks, I added some instructions.

Did you know that there's a `createdb` binary that ships with postgres?